frontends/bmd/src/app_mark_voter_card_invalid.test.tsx

```diff
@@ -89,11 +89,6 @@ describe('Mark Card Void when voter is idle too long', () => {
 
     // Idle reset timeout expires
     await advanceTimersAndPromises();
-
-    // Insert Card screen displays while card is read again.
-    screen.getByText('Insert Card');
-
-    // Card read again and now displays expired msg.
     await advanceTimersAndPromises();
     screen.getByText('Expired Card');
```

Comment:
Breaking UX change (though I consider it an improvement) - when a card expires from a timeout, we immediately show the Expired Card screen instead of first flashing the Insert Card screen.